lib/query/query_recurring_donations.rb

```diff
@@ -142,8 +154,9 @@ def self.for_export_enumerable(npo_id, query, chunk_limit=15000)
   end
 
   def self.get_chunk_of_export(npo_id, query, offset=nil, limit=nil, skip_header=false )
-    root_url = query[:root_url]
+    root_url = query[:root_url] || 'https://us.commitchange.com/'
```

Comment:
I had to do it because I couldn't find a way to properly get the root_url from there or from FailedRecurringDonationsReport :/

Reply:
That's fine. There's probably better ways but it's not worth looking into it right now.